content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md

```diff
@@ -0,0 +1,30 @@
+# Remove a model, version, or stage assignment
+
+To **remove a model** from the model registry, use the `Deprecate model` menu
+item in the 3-dot menu next to the model name in the models dashboard or details
+page. This action
+[creates a new Git tag in your Git repository to indicate that the model has been deprecated](https://mlem.ai/doc/gto/command-reference/deprecate#deprecating-an-artifact).
+
+You can also remove all of a project's models from Iterative Studio by deleting
+the project from your projects dashboard. This does not deprecate the model in
+the Git repository.
+
+To **deregister a model version**, use the `Deregister version` menu item in the
+3-dot menu next to the model name in the models dashboard or the
+`Deregister version` button on the model details page. This action creates a new
+Git tag in your Git repository to
+[indicate that the given model has been deregistered](https://mlem.ai/doc/gto/command-reference/deprecate#deregister-a-version).
+
+To **unassign a stage from a model version**, use the `Unassign stage` menu item
+in the 3-dot menu next to the model name in the models dashboard. You can also
+click on the the relevant stage assignment pill in the `Stages` section of the
+model details page to reveal the `Unassign stage` menu item. This action creates
+a new Git tag in your Git repository to
+[indicate that the given model version has been unassigned from the given stage](https://mlem.ai/doc/gto/command-reference/deprecate#unassigning-a-stage).
+
+Removing a Git tag from your Git repository also removes the model version or
```

Comment:
@aguschin Does it make sense to keep this paragraph or delete it? If we should keep it, it is complete and correct?

Reply:
@tapadipti, I'm not seeing this page in docs at all. Wasn't it deleted from the sidebar?

Reply:
@aguschin This is a new page created in this PR.

Reply:
Sorry, didn't see it first. You don't need to do force import. GH will call webhook and Studio BE will update this.

Reply:
Are screen shots a follow up item here? Feels like they would be way more useful than text.

Reply:
Yes, I'll raise a new PR with these screenshots as well as updates to other outdated ones.